.travis.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     os: linux
     arch: s390x
     jdk: openjdk11
-    script: mvn install -DskipTests -Dmaven.javadoc.skip=true
+    script: mvn install -DskipTests -Dmaven.javadoc.skip=true && mvn test
```

Comment:
Wouldn't be better just remove `-DskipTests`?

Reply:
Thanks @Frawless 

Removing `-DskipTests` will run integration tests for Bridge as well, which will require image `quay.io/strimzi/kafka:0.25.0-kafka-2.8.0`. Unfortunately this image does not support s390x at the moment. 

Once Strimzi Kafka Operator has fully s390x support, this image will support s390x and we can remove `-DskipTests` from `mvn install` command to run full test suite for Bridge.

Reply:
Hi @Frawless , I replaced `-DskipTests` with `-DskipITs` to exclude integration tests. It would be a better solution for now. Thanks for your suggestion!

Reply:
I see, ok